spin/cmds/pip.py

```diff
@@ -0,0 +1,34 @@
+import click
+
+from .util import run as _run
+
+
+@click.command()
+@click.option(
+    "-v", "--verbose", is_flag=True, help="Print detailed build and installation output"
+)
+@click.argument("-e", "--editable", is_flag=True, help="Install in editable mode")
+@click.argument("pip_args", nargs=-1)
+def install(pip_args, verbose=False, editable=True):
+    """💽 Install package and also build.
+
+    pip arguments are passed through in the same manner as you would pip e.g.:
+
+        spin install -- -r requirements.txt --editable false
```

Comment:
My point was that you probably don't want to do this. You want something like: `spin install -- --no-clean` or another option that makes sense

Reply:
Ah I see. sure that makes sense. I edited the example